app/src/main/java/com/github/orkest/View/Sharing.kt

```diff
@@ -13,11 +14,32 @@ import com.spotify.protocol.types.PlayerState
 /**
  * Activity used to test the Spotify API (DEMO ONLY)
  */
-class MainActivity2 : AppCompatActivity() {
+class Sharing : AppCompatActivity() {
     private var mSpotifyAppRemote: SpotifyAppRemote? = null
     override fun onCreate(savedInstanceState: Bundle?) {
+        when (intent?.action) {
+            Intent.ACTION_SEND -> {
+                if ("text/plain" == intent.type) {
+                    Log.d("Debug", "got text")
+                    handleSendText(intent) // Handle text being sent
+                } else if (intent.type?.startsWith("image/") == true) {
+                    Log.d("Debug", "got image")
+                    handleSendImage(intent) // Handle single image being sent
+                }
+            }
```

Comment:
For ACTION_SEND will we later need to implement a try catch if it throws exceptions? Maybe not for this Sprint but if necessary specify it for the next one